### PR TITLE
feat(vrack-services): dissociate vrack services from vrack

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/vrack-services`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14009
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

add vrack dissociation on vrack services overview page
commit bc4fc2000cb548ab5fcb65fdd0d536227ec1414f is already validated here https://github.com/ovh/manager/pull/11267 but I needed to cherry-pick it on this branch because it is not yet merged on master

## Related

- [ ] CMT-10506

### CODE REVIEW

File: packages/manager/apps/vrack-services/mock/setup.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const setupMocks = async () =>
   setupWorker(
     ...toMswHandlers(
       getConfig({
-        nbVs: 19,
+        nbVs: 20,
```
Comment (chipp972) [inline packages/manager/apps/vrack-services/mock/setup.ts:9]:
```suggestion
        nbVs: 19,
```

It's the setup used in dev mode but if we put 20, the create button is disabled

Comment (qpavy) [reply packages/manager/apps/vrack-services/mock/setup.ts:9]: oh Ok I will change that

File: packages/manager/apps/vrack-services/mock/vrack-services/get-vrack-services.json [hunk 2]
```diff
@@ -429,5 +429,51 @@
...
+    },
+    "currentTasks": [],
+    "id": "vrs-d18-qi7-5lk-is2",
+    "resourceStatus": "READY",
+    "targetSpec": {
+      "displayName": "SUB-test",
+      "subnets": [
+        {
+          "cidr": "10.0.0.0/24",
+          "displayName": null,
```
Comment (chipp972) [inline packages/manager/apps/vrack-services/mock/vrack-services/get-vrack-services.json:464]: This vrack services is not used I think

File: packages/manager/apps/vrack-services/src/components/DissociateVrackModal.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      </OsdsButton>
+      <OsdsButton
+        disabled={isLoading || undefined}
+        slot="actions"
+        type={ODS_BUTTON_TYPE.button}
+        variant={ODS_BUTTON_VARIANT.flat}
+        color={ODS_THEME_COLOR_INTENT.error}
+        {...handleClick(() => {
+          dissociateVs();
+          trackClick({
```
Comment (chipp972) [inline packages/manager/apps/vrack-services/src/components/DissociateVrackModal.tsx:132]: send tracking click if there are dataTrackingPath and dataTrackingConfirmValue

File: packages/manager/apps/vrack-services/src/components/DissociateVrackModal.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  dataTrackingPath?: string;
+  dataTrackingConfirmValue?: string;
+  dataTrackingCancelValue?: string;
+  vrackId: string;
+  vrackServicesId: string;
+};
+
+export const DissociateVrackModal: React.FC<DissociateVrackModalProps> = ({
+  headline,
+  description,
```
Comment (chipp972) [inline packages/manager/apps/vrack-services/src/components/DissociateVrackModal.tsx:44]: We could put the headline and description directly in the modal, don't we ?

Those labels won't change even if we put this modal in a different page

File: packages/manager/apps/vrack-services/mock/vrack-services/get-vrack-services.json (hunk 2 above)
Comment (qpavy) [reply packages/manager/apps/vrack-services/mock/vrack-services/get-vrack-services.json:464]: it is used here => https://github.com/ovh/manager/pull/11293/files#diff-2a48f24cf860380cfed45c7d55c10c1207e6c6904cfe0679d99ff5d26d6d36c4R133

Comment (chipp972) [reply packages/manager/apps/vrack-services/mock/vrack-services/get-vrack-services.json:464]: ok good

**Review by chipp972 — DISMISSED** [review]

**Review by aboungnaseng-ovhcloud — DISMISSED** [review]

**Review by chipp972 — APPROVED** [review]

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

**Pull request merged**